{
  "version": "2.1.0",
  "$schema": "https://raw.githubusercontent.com/oasis-tcs/sarif-spec/master/Schemata/sarif-schema-2.1.0.json",
  "runs": [
    {
      "tool": {
        "driver": {
          "informationUri": "https://docs.jfrog-applications.jfrog.io/jfrog-security-features/sca",
          "name": "JFrog Xray SCA",
          "rules": [
            {
              "id": "CVE-2024-4741_libopenssl1_1_0:1.1.1l-150500.17.28.2",
              "shortDescription": {
                "text": "[CVE-2024-4741] libopenssl1_1 0:1.1.1l-150500.17.28.2"
              },
              "help": {
                "text": "Security update for openssl-1_1 (Important)\nSecurity update for openssl-3 (Important)",
                "markdown": "| Severity Score | Direct Dependencies | Fixed Versions     |\n| :---:        |    :----:   |          :---: |\n| 8.1      | `sha256__c202c2f3154b7a3480f4215cb446f9a1ef43affe8ca728bf292317d31cc260ea.tar `       | No fix available   |"
              },
              "properties": {
                "security-severity": "8.1"
              }
            },
            {
              "id": "CVE-2024-24806_libuv1_0:1.44.2-150500.3.2.1",
              "shortDescription": {
                "text": "[CVE-2024-24806] libuv1 0:1.44.2-150500.3.2.1"
              },
              "help": {
                "text": "Security update for nodejs12 (Important)\nSecurity update for nodejs14 (Important)\nSecurity update for nodejs16 (Important)\nSecurity update for nodejs18 (Important)\nSecurity update for nodejs20 (Important)",
                "markdown": "| Severity Score | Direct Dependencies | Fixed Versions     |\n| :---:        |    :----:   |          :---: |\n| 5.8      | `sha256__c202c2f3154b7a3480f4215cb446f9a1ef43affe8ca728bf292317d31cc260ea.tar `       | No fix available   |"
              },
              "properties": {
                "security-severity": "5.8"
              }
            },
            {
              "id": "CVE-2024-0450_python3_0:3.6.15-150300.10.60.1",
              "shortDescription": {
                "text": "[CVE-2024-0450] python3 0:3.6.15-150300.10.60.1"
              },
              "help": {
                "text": "Security update for python (Moderate)\nSecurity update for python3 (Important)\nSecurity update for python3 (Moderate)\nSecurity update for python310 (Important)\nSecurity update for python311 (Important)\nSecurity update for python36 (Important)\nSecurity update for python39 (Important)",
                "markdown": "| Severity Score | Direct Dependencies | Fixed Versions     |\n| :---:        |    :----:   |          :---: |\n| 6.2      | `sha256__c202c2f3154b7a3480f4215cb446f9a1ef43affe8ca728bf292317d31cc260ea.tar `       | No fix available   |"
              },
              "properties": {
                "security-severity": "6.2"
              }
            },
            {
              "id": "CVE-2024-24557_github.com/docker/docker_20.10.27+incompatible",
              "shortDescription": {
                "text": "[CVE-2024-24557] github.com/docker/docker 20.10.27+incompatible"
              },
              "help": {
                "text": "Moby is an open-source project created by Docker to enable software containerization. The classic builder cache system is prone to cache poisoning if the image is built FROM scratch. Also, changes to some instructions (most important being HEALTHCHECK and ONBUILD) would not cause a cache miss. An attacker with the knowledge of the Dockerfile someone is using could poison their cache by making them pull a specially crafted image that would be considered as a valid cache candidate for some build steps. 23.0+ users are only affected if they explicitly opted out of Buildkit (DOCKER_BUILDKIT=0 environment variable) or are using the /build API endpoint. All users on versions older than 23.0 could be impacted. Image build API endpoint (/build) and ImageBuild function from github.com/docker/docker/client is also affected as it the uses classic builder by default. Patches are included in 24.0.9 and 25.0.2 releases.",
                "markdown": "| Severity Score | Direct Dependencies | Fixed Versions     |\n| :---:        |    :----:   |          :---: |\n| 7.8      | `sha256__a4a91401fa779ddbf21deda237e38b72e8d8bfc38842fef8753d0b877e305d45.tar `       | [24.0.9], [25.0.2]   |"
              },
              "properties": {
                "security-severity": "7.8"
              }
            },
            {
              "id": "CVE-2018-20225_pip_20.0.2",
              "shortDescription": {
                "text": "[CVE-2018-20225] pip 20.0.2"
              },
              "help": {
                "text": "An issue was discovered in pip (all versions) because it installs the version with the highest version number, even if the user had intended to obtain a private package from a private index. This only affects use of the --extra-index-url option, and exploitation requires that the package does not already exist in the public index (and thus the attacker can put the package there with an arbitrary version number). NOTE: it has been reported that this is intended functionality and the user is responsible for using --extra-index-url securely",
                "markdown": "| Severity Score | Direct Dependencies | Fixed Versions     |\n| :---:        |    :----:   |          :---: |\n| 7.8      | `sha256__c202c2f3154b7a3480f4215cb446f9a1ef43affe8ca728bf292317d31cc260ea.tar `       | No fix available   |"
              },
              "properties": {
                "security-severity": "7.8"
              }
            },
            {
              "id": "CVE-2024-24577_git-core_0:2.35.3-150300.10.39.1",
              "shortDescription": {
                "text": "[CVE-2024-24577] git-core 0:2.35.3-150300.10.39.1"
              },
              "help": {
                "text": "CVE-2024-24577",
                "markdown": "| Severity Score | Direct Dependencies | Fixed Versions     |\n| :---:        |    :----:   |          :---: |\n| 8.6      | `sha256__c202c2f3154b7a3480f4215cb446f9a1ef43affe8ca728bf292317d31cc260ea.tar `       | No fix available   |"
              },
              "properties": {
                "security-severity": "8.6"
              }
            },
            {
              "id": "CVE-2023-52425_python3_0:3.6.15-150300.10.60.1",
              "shortDescription": {
                "text": "[CVE-2023-52425] python3 0:3.6.15-150300.10.60.1"
              },
              "help": {
                "text": "Security update for expat (Important)\nSecurity update for python (Moderate)\nSecurity update for python3 (Important)\nSecurity update for python3 (Moderate)\nSecurity update for python310 (Important)\nSecurity update for python310 (Moderate)\nSecurity update for python311 (Important)\nSecurity update for python36 (Important)\nSecurity update for python39 (Important)",
                "markdown": "| Severity Score | Direct Dependencies | Fixed Versions     |\n| :---:        |    :----:   |          :---: |\n| 5.5      | `sha256__c202c2f3154b7a3480f4215cb446f9a1ef43affe8ca728bf292317d31cc260ea.tar `       | No fix available   |"
              },
              "properties": {
                "security-severity": "5.5"
              }
            },
            {
              "id": "CVE-2024-4741_openssl-1_1_0:1.1.1l-150500.17.28.2",
              "shortDescription": {
                "text": "[CVE-2024-4741] openssl-1_1 0:1.1.1l-150500.17.28.2"
              },
              "help": {
                "text": "Security update for openssl-1_1 (Important)\nSecurity update for openssl-3 (Important)",
                "markdown": "| Severity Score | Direct Dependencies | Fixed Versions     |\n| :---:        |    :----:   |          :---: |\n| 8.1      | `sha256__c202c2f3154b7a3480f4215cb446f9a1ef43affe8ca728bf292317d31cc260ea.tar `       | No fix available   |"
              },
              "properties": {
                "security-severity": "8.1"
              }
            },
            {
              "id": "CVE-2023-5408_k8s.io/kubernetes_1.27.10",
              "shortDescription": {
                "text": "[CVE-2023-5408] k8s.io/kubernetes 1.27.10"
              },
              "help": {
                "text": "A privilege escalation flaw was found in the node restriction admission plugin of the kubernetes api server of OpenShift. A remote attacker who modifies the node role label could steer workloads from the control plane and etcd nodes onto different worker nodes and gain broader access to the cluster.",
                "markdown": "| Severity Score | Direct Dependencies | Fixed Versions     |\n| :---:        |    :----:   |          :---: |\n| 7.2      | `sha256__5bc13c921dc05e49c915f3073ec83a13074e1711636b35c1f8b3b6bacdbda38e.tar `       | No fix available   |"
              },
              "properties": {
                "security-severity": "7.2"
              }
            },
            {
              "id": "CVE-2023-45288_golang.org/x/net_0.19.0",
              "shortDescription": {
                "text": "[CVE-2023-45288] golang.org/x/net 0.19.0"
              },
              "help": {
                "text": "An attacker may cause an HTTP/2 endpoint to read arbitrary amounts of header data by sending an excessive number of CONTINUATION frames. Maintaining HPACK state requires parsing and processing all HEADERS and CONTINUATION frames on a connection. When a request's headers exceed MaxHeaderBytes, no memory is allocated to store the excess headers, but they are still parsed. This permits an attacker to cause an HTTP/2 endpoint to read arbitrary amounts of header data, all associated with a request which is going to be rejected. These headers can include Huffman-encoded data which is significantly more expensive for the receiver to decode than for an attacker to send. The fix sets a limit on the amount of excess header frames we will process before closing a connection.",
                "markdown": "| Severity Score | Direct Dependencies | Fixed Versions     |\n| :---:        |    :----:   |          :---: |\n| 5.3      | `sha256__5bc13c921dc05e49c915f3073ec83a13074e1711636b35c1f8b3b6bacdbda38e.tar `       | [0.23.0]   |"
              },
              "properties": {
                "security-severity": "5.3"
              }
            },
            {
              "id": "CVE-2024-29018_github.com/docker/docker_20.10.27+incompatible",
              "shortDescription": {
                "text": "[CVE-2024-29018] github.com/docker/docker 20.10.27+incompatible"
              },
              "help": {
                "text": "Moby is an open source container framework that is a key component of Docker Engine, Docker Desktop, and other distributions of container tooling or runtimes. Moby's networking implementation allows for many networks, each with their own IP address range and gateway, to be defined. This feature is frequently referred to as custom networks, as each network can have a different driver, set of parameters and thus behaviors. When creating a network, the `--internal` flag is used to designate a network as _internal_. The `internal` attribute in a docker-compose.yml file may also be used to mark a network _internal_, and other API clients may specify the `internal` parameter as well.\n\nWhen containers with networking are created, they are assigned unique network interfaces and IP addresses. The host serves as a router for non-internal networks, with a gateway IP that provides SNAT/DNAT to/from container IPs.\n\nContainers on an internal network may communicate between each other, but are precluded from communicating with any networks the host has access to (LAN or WAN) as no default route is configured, and firewall rules are set up to drop all outgoing traffic. Communication with the gateway IP address (and thus appropriately configured host services) is possible, and the host may communicate with any container IP directly.\n\nIn addition to configuring the Linux kernel's various networking features to enable container networking, `dockerd` directly provides some services to container networks. Principal among these is serving as a resolver, enabling service discovery, and resolution of names from an upstream resolver.\n\nWhen a DNS request for a name that does not correspond to a container is received, the request is forwarded to the configured upstream resolver. This request is made from the container's network namespace: the level of access and routing of traffic is the same as if the request was made by the container itself.\n\nAs a consequence of this design, containers solely attached to an internal network will ...",
                "markdown": "| Severity Score | Direct Dependencies | Fixed Versions     |\n| :---:        |    :----:   |          :---: |\n| 5.9      | `sha256__a4a91401fa779ddbf21deda237e38b72e8d8bfc38842fef8753d0b877e305d45.tar `       | [23.0.11], [25.0.5], [26.0.0-rc3]   |"
              },
              "properties": {
                "security-severity": "5.9"
              }
            },
            {
              "id": "CVE-2023-2253_github.com/docker/distribution_2.8.1+incompatible",
              "shortDescription": {
                "text": "[CVE-2023-2253] github.com/docker/distribution 2.8.1+incompatible"
              },
              "help": {
                "text": "A flaw was found in the `/v2/_catalog` endpoint in distribution/distribution, which accepts a parameter to control the maximum number of records returned (query string: `n`). This vulnerability allows a malicious user to submit an unreasonably large value for `n,` causing the allocation of a massive string array, possibly causing a denial of service through excessive use of memory.",
                "markdown": "| Severity Score | Direct Dependencies | Fixed Versions     |\n| :---:        |    :----:   |          :---: |\n| 6.5      | `sha256__5bc13c921dc05e49c915f3073ec83a13074e1711636b35c1f8b3b6bacdbda38e.tar `       | [2.8.2-beta.1]   |"
              },
              "properties": {
                "security-severity": "6.5"
              }
            },
            {
              "id": "CVE-2022-40897_setuptools_44.1.1",
              "shortDescription": {
                "text": "[CVE-2022-40897] setuptools 44.1.1"
              },
              "help": {
                "text": "Python Packaging Authority (PyPA) setuptools before 65.5.1 allows remote attackers to cause a denial of service via HTML in a crafted package or custom PackageIndex page. There is a Regular Expression Denial of Service (ReDoS) in package_index.py.",
                "markdown": "| Severity Score | Direct Dependencies | Fixed Versions     |\n| :---:        |    :----:   |          :---: |\n| 5.9      | `sha256__c202c2f3154b7a3480f4215cb446f9a1ef43affe8ca728bf292317d31cc260ea.tar `       | [65.5.1]   |"
              },
              "properties": {
                "security-severity": "5.9"
              }
            },
            {
              "id": "CVE-2024-24786_google.golang.org/protobuf_1.31.0",
              "shortDescription": {
                "text": "[CVE-2024-24786] google.golang.org/protobuf 1.31.0"
              },
              "help": {
                "text": "The protojson.Unmarshal function can enter an infinite loop when unmarshaling certain forms of invalid JSON. This condition can occur when unmarshaling into a message which contains a google.protobuf.Any value, or when the UnmarshalOptions.DiscardUnknown option is set.",
                "markdown": "| Severity Score | Direct Dependencies | Fixed Versions     |\n| :---:        |    :----:   |          :---: |\n| 0.0      | `sha256__5bc13c921dc05e49c915f3073ec83a13074e1711636b35c1f8b3b6bacdbda38e.tar `       | [1.33.0]   |"
              },
              "properties": {
                "security-severity": "0.0"
              }
            },
            {
              "id": "CVE-2021-3572_pip_20.0.2",
              "shortDescription": {
                "text": "[CVE-2021-3572] pip 20.0.2"
              },
              "help": {
                "text": "A flaw was found in python-pip in the way it handled Unicode separators in git references. A remote attacker could possibly use this issue to install a different revision on a repository. The highest threat from this vulnerability is to data integrity. This is fixed in python-pip version 21.1.",
                "markdown": "| Severity Score | Direct Dependencies | Fixed Versions     |\n| :---:        |    :----:   |          :---: |\n| 5.7      | `sha256__c202c2f3154b7a3480f4215cb446f9a1ef43affe8ca728bf292317d31cc260ea.tar `       | [21.1]   |"
              },
              "properties": {
                "security-severity": "5.7"
              }
            },
            {
              "id": "CVE-2023-45288_github.com/golang/go_1.20.13",
              "shortDescription": {
                "text": "[CVE-2023-45288] github.com/golang/go 1.20.13"
              },
              "help": {
                "text": "An attacker may cause an HTTP/2 endpoint to read arbitrary amounts of header data by sending an excessive number of CONTINUATION frames. Maintaining HPACK state requires parsing and processing all HEADERS and CONTINUATION frames on a connection. When a request's headers exceed MaxHeaderBytes, no memory is allocated to store the excess headers, but they are still parsed. This permits an attacker to cause an HTTP/2 endpoint to read arbitrary amounts of header data, all associated with a request which is going to be rejected. These headers can include Huffman-encoded data which is significantly more expensive for the receiver to decode than for an attacker to send. The fix sets a limit on the amount of excess header frames we will process before closing a connection.",
                "markdown": "| Severity Score | Direct Dependencies | Fixed Versions     |\n| :---:        |    :----:   |          :---: |\n| 5.3      | `sha256__5bc13c921dc05e49c915f3073ec83a13074e1711636b35c1f8b3b6bacdbda38e.tar `       | [1.21.9], [1.22.2]   |"
              },
              "properties": {
                "security-severity": "5.3"
              }
            },
            {
              "id": "CVE-2024-0397_python3_0:3.6.15-150300.10.60.1",
              "shortDescription": {
                "text": "[CVE-2024-0397] python3 0:3.6.15-150300.10.60.1"
              },
              "help": {
                "text": "CVE-2024-0397",
                "markdown": "| Severity Score | Direct Dependencies | Fixed Versions     |\n| :---:        |    :----:   |          :---: |\n| 4.8      | `sha256__c202c2f3154b7a3480f4215cb446f9a1ef43affe8ca728bf292317d31cc260ea.tar `       | No fix available   |"
              },
              "properties": {
                "security-severity": "4.8"
              }
            },
            {
              "id": "CVE-2023-52426_libexpat1_0:2.4.4-150400.3.17.1",
              "shortDescription": {
                "text": "[CVE-2023-52426] libexpat1 0:2.4.4-150400.3.17.1"
              },
              "help": {
                "text": "CVE-2023-52426",
                "markdown": "| Severity Score | Direct Dependencies | Fixed Versions     |\n| :---:        |    :----:   |          :---: |\n| 5.5      | `sha256__c202c2f3154b7a3480f4215cb446f9a1ef43affe8ca728bf292317d31cc260ea.tar `       | No fix available   |"
              },
              "properties": {
                "security-severity": "5.5"
              }
            },
            {
              "id": "CVE-2019-13225_libonig4_0:6.7.0-150000.3.3.1",
              "shortDescription": {
                "text": "[CVE-2019-13225] libonig4 0:6.7.0-150000.3.3.1"
              },
              "help": {
                "text": "libonig5-6.9.8-2.8 on GA media (Moderate)",
                "markdown": "| Severity Score | Direct Dependencies | Fixed Versions     |\n| :---:        |    :----:   |          :---: |\n| 6.5      | `sha256__c202c2f3154b7a3480f4215cb446f9a1ef43affe8ca728bf292317d31cc260ea.tar `       | No fix available   |"
              },
              "properties": {
                "security-severity": "6.5"
              }
            },
            {
              "id": "CVE-2013-4235_shadow_0:4.8.1-150400.10.15.1",
              "shortDescription": {
                "text": "[CVE-2013-4235] shadow 0:4.8.1-150400.10.15.1"
              },
              "help": {
                "text": "libsubid4-4.15.1-1.1 on GA media (Moderate)",
                "markdown": "| Severity Score | Direct Dependencies | Fixed Versions     |\n| :---:        |    :----:   |          :---: |\n| 6.7      | `sha256__c202c2f3154b7a3480f4215cb446f9a1ef43affe8ca728bf292317d31cc260ea.tar `       | No fix available   |"
              },
              "properties": {
                "security-severity": "6.7"
              }
            },
            {
              "id": "CVE-2013-4235_login_defs_0:4.8.1-150400.10.15.1",
              "shortDescription": {
                "text": "[CVE-2013-4235] login_defs 0:4.8.1-150400.10.15.1"
              },
              "help": {
                "text": "libsubid4-4.15.1-1.1 on GA media (Moderate)",
                "markdown": "| Severity Score | Direct Dependencies | Fixed Versions     |\n| :---:        |    :----:   |          :---: |\n| 6.7      | `sha256__c202c2f3154b7a3480f4215cb446f9a1ef43affe8ca728bf292317d31cc260ea.tar `       | No fix available   |"
              },
              "properties": {
                "security-severity": "6.7"
              }
            },
            {
              "id": "CVE-2023-5752_pip_20.0.2",
              "shortDescription": {
                "text": "[CVE-2023-5752] pip 20.0.2"
              },
              "help": {
                "text": "When installing a package from a Mercurial VCS URL  (ie \"pip install \nhg+...\") with pip prior to v23.3, the specified Mercurial revision could\n be used to inject arbitrary configuration options to the \"hg clone\" \ncall (ie \"--config\"). Controlling the Mercurial configuration can modify\n how and which repository is installed. This vulnerability does not \naffect users who aren't installing from Mercurial.",
                "markdown": "| Severity Score | Direct Dependencies | Fixed Versions     |\n| :---:        |    :----:   |          :---: |\n| 3.3      | `sha256__c202c2f3154b7a3480f4215cb446f9a1ef43affe8ca728bf292317d31cc260ea.tar `       | [23.3]   |"
              },
              "properties": {
                "security-severity": "3.3"
              }
            },
            {
              "id": "CVE-2024-34459_libxml2-2_0:2.10.3-150500.5.14.1",
              "shortDescription": {
                "text": "[CVE-2024-34459] libxml2-2 0:2.10.3-150500.5.14.1"
              },
              "help": {
                "text": "CVE-2024-34459",
                "markdown": "| Severity Score | Direct Dependencies | Fixed Versions     |\n| :---:        |    :----:   |          :---: |\n| 3.3      | `sha256__c202c2f3154b7a3480f4215cb446f9a1ef43affe8ca728bf292317d31cc260ea.tar `       | No fix available   |"
              },
              "properties": {
                "security-severity": "3.3"
              }
            },
            {
              "id": "CVE-2024-3177_k8s.io/kubernetes_1.27.10",
              "shortDescription": {
                "text": "[CVE-2024-3177] k8s.io/kubernetes 1.27.10"
              },
              "help": {
                "text": "A security issue was discovered in Kubernetes where users may be able to launch containers that bypass the mountable secrets policy enforced by the ServiceAccount admission plugin when using containers, init containers, and ephemeral containers with the envFrom field populated. The policy ensures pods running with a service account may only reference secrets specified in the service account’s secrets field. Kubernetes clusters are only affected if the ServiceAccount admission plugin and the kubernetes.io/enforce-mountable-secrets annotation are used together with containers, init containers, and ephemeral containers with the envFrom field populated.",
                "markdown": "| Severity Score | Direct Dependencies | Fixed Versions     |\n| :---:        |    :----:   |          :---: |\n| 2.7      | `sha256__5bc13c921dc05e49c915f3073ec83a13074e1711636b35c1f8b3b6bacdbda38e.tar `       | [1.27.13], [1.28.9], [1.29.4]   |"
              },
              "properties": {
                "security-severity": "2.7"
              }
            },
            {
              "id": "CVE-2024-3177_k8s.io/kubernetes_1.29.3",
              "shortDescription": {
                "text": "[CVE-2024-3177] k8s.io/kubernetes 1.29.3"
              },
              "help": {
                "text": "A security issue was discovered in Kubernetes where users may be able to launch containers that bypass the mountable secrets policy enforced by the ServiceAccount admission plugin when using containers, init containers, and ephemeral containers with the envFrom field populated. The policy ensures pods running with a service account may only reference secrets specified in the service account’s secrets field. Kubernetes clusters are only affected if the ServiceAccount admission plugin and the kubernetes.io/enforce-mountable-secrets annotation are used together with containers, init containers, and ephemeral containers with the envFrom field populated.",
                "markdown": "| Severity Score | Direct Dependencies | Fixed Versions     |\n| :---:        |    :----:   |          :---: |\n| 2.7      | `sha256__a4a91401fa779ddbf21deda237e38b72e8d8bfc38842fef8753d0b877e305d45.tar `       | [1.27.13], [1.28.9], [1.29.4]   |"
              },
              "properties": {
                "security-severity": "2.7"
              }
            },
            {
              "id": "CVE-2024-4032_python3_0:3.6.15-150300.10.60.1",
              "shortDescription": {
                "text": "[CVE-2024-4032] python3 0:3.6.15-150300.10.60.1"
              },
              "help": {
                "text": "CVE-2024-4032",
                "markdown": "| Severity Score | Direct Dependencies | Fixed Versions     |\n| :---:        |    :----:   |          :---: |\n| 3.7      | `sha256__c202c2f3154b7a3480f4215cb446f9a1ef43affe8ca728bf292317d31cc260ea.tar `       | No fix available   |"
              },
              "properties": {
                "security-severity": "3.7"
              }
            }
          ],
          "version": "3.96.1"
        }
      },
      "results": [
        {
          "ruleId": "CVE-2024-4741_libopenssl1_1_0:1.1.1l-150500.17.28.2",
          "ruleIndex": 0,
          "level": "error",
          "message": {
            "text": "[CVE-2024-4741] sha256__c202c2f3154b7a3480f4215cb446f9a1ef43affe8ca728bf292317d31cc260ea.tar "
          },
          "locations": [
            {
              "physicalLocation": {
                "artifactLocation": {
                  "uri": "file://Package-Descriptor"
                }
              }
            }
          ]
        },
        {
          "ruleId": "CVE-2024-24806_libuv1_0:1.44.2-150500.3.2.1",
          "ruleIndex": 1,
          "level": "error",
          "message": {
            "text": "[CVE-2024-24806] sha256__c202c2f3154b7a3480f4215cb446f9a1ef43affe8ca728bf292317d31cc260ea.tar "
          },
          "locations": [
            {
              "physicalLocation": {
                "artifactLocation": {
                  "uri": "file://Package-Descriptor"
                }
              }
            }
          ]
        },
        {
          "ruleId": "CVE-2024-0450_python3_0:3.6.15-150300.10.60.1",
          "ruleIndex": 2,
          "level": "error",
          "message": {
            "text": "[CVE-2024-0450] sha256__c202c2f3154b7a3480f4215cb446f9a1ef43affe8ca728bf292317d31cc260ea.tar "
          },
          "locations": [
            {
              "physicalLocation": {
                "artifactLocation": {
                  "uri": "file://Package-Descriptor"
                }
              }
            }
          ]
        },
        {
          "ruleId": "CVE-2024-24557_github.com/docker/docker_20.10.27+incompatible",
          "ruleIndex": 3,
          "level": "error",
          "message": {
            "text": "[CVE-2024-24557] sha256__a4a91401fa779ddbf21deda237e38b72e8d8bfc38842fef8753d0b877e305d45.tar "
          },
          "locations": [
            {
              "physicalLocation": {
                "artifactLocation": {
                  "uri": "file://Package-Descriptor"
                }
              }
            }
          ]
        },
        {
          "ruleId": "CVE-2018-20225_pip_20.0.2",
          "ruleIndex": 4,
          "level": "error",
          "message": {
            "text": "[CVE-2018-20225] sha256__c202c2f3154b7a3480f4215cb446f9a1ef43affe8ca728bf292317d31cc260ea.tar "
          },
          "locations": [
            {
              "physicalLocation": {
                "artifactLocation": {
                  "uri": "file://Package-Descriptor"
                }
              }
            }
          ]
        },
        {
          "ruleId": "CVE-2024-24577_git-core_0:2.35.3-150300.10.39.1",
          "ruleIndex": 5,
          "level": "error",
          "message": {
            "text": "[CVE-2024-24577] sha256__c202c2f3154b7a3480f4215cb446f9a1ef43affe8ca728bf292317d31cc260ea.tar "
          },
          "locations": [
            {
              "physicalLocation": {
                "artifactLocation": {
                  "uri": "file://Package-Descriptor"
                }
              }
            }
          ]
        },
        {
          "ruleId": "CVE-2023-52425_python3_0:3.6.15-150300.10.60.1",
          "ruleIndex": 6,
          "level": "error",
          "message": {
            "text": "[CVE-2023-52425] sha256__c202c2f3154b7a3480f4215cb446f9a1ef43affe8ca728bf292317d31cc260ea.tar "
          },
          "locations": [
            {
              "physicalLocation": {
                "artifactLocation": {
                  "uri": "file://Package-Descriptor"
                }
              }
            }
          ]
        },
        {
          "ruleId": "CVE-2024-4741_openssl-1_1_0:1.1.1l-150500.17.28.2",
          "ruleIndex": 7,
          "level": "error",
          "message": {
            "text": "[CVE-2024-4741] sha256__c202c2f3154b7a3480f4215cb446f9a1ef43affe8ca728bf292317d31cc260ea.tar "
          },
          "locations": [
            {
              "physicalLocation": {
                "artifactLocation": {
                  "uri": "file://Package-Descriptor"
                }
              }
            }
          ]
        },
        {
          "ruleId": "CVE-2023-5408_k8s.io/kubernetes_1.27.10",
          "ruleIndex": 8,
          "level": "error",
          "message": {
            "text": "[CVE-2023-5408] sha256__5bc13c921dc05e49c915f3073ec83a13074e1711636b35c1f8b3b6bacdbda38e.tar "
          },
          "locations": [
            {
              "physicalLocation": {
                "artifactLocation": {
                  "uri": "file://Package-Descriptor"
                }
              }
            }
          ]
        },
        {
          "ruleId": "CVE-2023-45288_golang.org/x/net_0.19.0",
          "ruleIndex": 9,
          "level": "warning",
          "message": {
            "text": "[CVE-2023-45288] sha256__5bc13c921dc05e49c915f3073ec83a13074e1711636b35c1f8b3b6bacdbda38e.tar "
          },
          "locations": [
            {
              "physicalLocation": {
                "artifactLocation": {
                  "uri": "file://Package-Descriptor"
                }
              }
            }
          ]
        },
        {
          "ruleId": "CVE-2024-29018_github.com/docker/docker_20.10.27+incompatible",
          "ruleIndex": 10,
          "level": "warning",
          "message": {
            "text": "[CVE-2024-29018] sha256__a4a91401fa779ddbf21deda237e38b72e8d8bfc38842fef8753d0b877e305d45.tar "
          },
          "locations": [
            {
              "physicalLocation": {
                "artifactLocation": {
                  "uri": "file://Package-Descriptor"
                }
              }
            }
          ]
        },
        {
          "ruleId": "CVE-2023-2253_github.com/docker/distribution_2.8.1+incompatible",
          "ruleIndex": 11,
          "level": "warning",
          "message": {
            "text": "[CVE-2023-2253] sha256__5bc13c921dc05e49c915f3073ec83a13074e1711636b35c1f8b3b6bacdbda38e.tar "
          },
          "locations": [
            {
              "physicalLocation": {
                "artifactLocation": {
                  "uri": "file://Package-Descriptor"
                }
              }
            }
          ]
        },
        {
          "ruleId": "CVE-2022-40897_setuptools_44.1.1",
          "ruleIndex": 12,
          "level": "warning",
          "message": {
            "text": "[CVE-2022-40897] sha256__c202c2f3154b7a3480f4215cb446f9a1ef43affe8ca728bf292317d31cc260ea.tar "
          },
          "locations": [
            {
              "physicalLocation": {
                "artifactLocation": {
                  "uri": "file://Package-Descriptor"
                }
              }
            }
          ]
        },
        {
          "ruleId": "CVE-2024-24786_google.golang.org/protobuf_1.31.0",
          "ruleIndex": 13,
          "level": "warning",
          "message": {
            "text": "[CVE-2024-24786] sha256__5bc13c921dc05e49c915f3073ec83a13074e1711636b35c1f8b3b6bacdbda38e.tar "
          },
          "locations": [
            {
              "physicalLocation": {
                "artifactLocation": {
                  "uri": "file://Package-Descriptor"
                }
              }
            }
          ]
        },
        {
          "ruleId": "CVE-2021-3572_pip_20.0.2",
          "ruleIndex": 14,
          "level": "warning",
          "message": {
            "text": "[CVE-2021-3572] sha256__c202c2f3154b7a3480f4215cb446f9a1ef43affe8ca728bf292317d31cc260ea.tar "
          },
          "locations": [
            {
              "physicalLocation": {
                "artifactLocation": {
                  "uri": "file://Package-Descriptor"
                }
              }
            }
          ]
        },
        {
          "ruleId": "CVE-2023-45288_github.com/golang/go_1.20.13",
          "ruleIndex": 15,
          "level": "warning",
          "message": {
            "text": "[CVE-2023-45288] sha256__5bc13c921dc05e49c915f3073ec83a13074e1711636b35c1f8b3b6bacdbda38e.tar "
          },
          "locations": [
            {
              "physicalLocation": {
                "artifactLocation": {
                  "uri": "file://Package-Descriptor"
                }
              }
            }
          ]
        },
        {
          "ruleId": "CVE-2024-0397_python3_0:3.6.15-150300.10.60.1",
          "ruleIndex": 16,
          "level": "warning",
          "message": {
            "text": "[CVE-2024-0397] sha256__c202c2f3154b7a3480f4215cb446f9a1ef43affe8ca728bf292317d31cc260ea.tar "
          },
          "locations": [
            {
              "physicalLocation": {
                "artifactLocation": {
                  "uri": "file://Package-Descriptor"
                }
              }
            }
          ]
        },
        {
          "ruleId": "CVE-2023-52426_libexpat1_0:2.4.4-150400.3.17.1",
          "ruleIndex": 17,
          "level": "warning",
          "message": {
            "text": "[CVE-2023-52426] sha256__c202c2f3154b7a3480f4215cb446f9a1ef43affe8ca728bf292317d31cc260ea.tar "
          },
          "locations": [
            {
              "physicalLocation": {
                "artifactLocation": {
                  "uri": "file://Package-Descriptor"
                }
              }
            }
          ]
        },
        {
          "ruleId": "CVE-2019-13225_libonig4_0:6.7.0-150000.3.3.1",
          "ruleIndex": 18,
          "level": "warning",
          "message": {
            "text": "[CVE-2019-13225] sha256__c202c2f3154b7a3480f4215cb446f9a1ef43affe8ca728bf292317d31cc260ea.tar "
          },
          "locations": [
            {
              "physicalLocation": {
                "artifactLocation": {
                  "uri": "file://Package-Descriptor"
                }
              }
            }
          ]
        },
        {
          "ruleId": "CVE-2013-4235_shadow_0:4.8.1-150400.10.15.1",
          "ruleIndex": 19,
          "level": "warning",
          "message": {
            "text": "[CVE-2013-4235] sha256__c202c2f3154b7a3480f4215cb446f9a1ef43affe8ca728bf292317d31cc260ea.tar "
          },
          "locations": [
            {
              "physicalLocation": {
                "artifactLocation": {
                  "uri": "file://Package-Descriptor"
                }
              }
            }
          ]
        },
        {
          "ruleId": "CVE-2013-4235_login_defs_0:4.8.1-150400.10.15.1",
          "ruleIndex": 20,
          "level": "warning",
          "message": {
            "text": "[CVE-2013-4235] sha256__c202c2f3154b7a3480f4215cb446f9a1ef43affe8ca728bf292317d31cc260ea.tar "
          },
          "locations": [
            {
              "physicalLocation": {
                "artifactLocation": {
                  "uri": "file://Package-Descriptor"
                }
              }
            }
          ]
        },
        {
          "ruleId": "CVE-2023-5752_pip_20.0.2",
          "ruleIndex": 21,
          "level": "note",
          "message": {
            "text": "[CVE-2023-5752] sha256__c202c2f3154b7a3480f4215cb446f9a1ef43affe8ca728bf292317d31cc260ea.tar "
          },
          "locations": [
            {
              "physicalLocation": {
                "artifactLocation": {
                  "uri": "file://Package-Descriptor"
                }
              }
            }
          ]
        },
        {
          "ruleId": "CVE-2024-34459_libxml2-2_0:2.10.3-150500.5.14.1",
          "ruleIndex": 22,
          "level": "note",
          "message": {
            "text": "[CVE-2024-34459] sha256__c202c2f3154b7a3480f4215cb446f9a1ef43affe8ca728bf292317d31cc260ea.tar "
          },
          "locations": [
            {
              "physicalLocation": {
                "artifactLocation": {
                  "uri": "file://Package-Descriptor"
                }
              }
            }
          ]
        },
        {
          "ruleId": "CVE-2024-3177_k8s.io/kubernetes_1.27.10",
          "ruleIndex": 23,
          "level": "note",
          "message": {
            "text": "[CVE-2024-3177] sha256__5bc13c921dc05e49c915f3073ec83a13074e1711636b35c1f8b3b6bacdbda38e.tar "
          },
          "locations": [
            {
              "physicalLocation": {
                "artifactLocation": {
                  "uri": "file://Package-Descriptor"
                }
              }
            }
          ]
        },
        {
          "ruleId": "CVE-2024-3177_k8s.io/kubernetes_1.29.3",
          "ruleIndex": 24,
          "level": "note",
          "message": {
            "text": "[CVE-2024-3177] sha256__a4a91401fa779ddbf21deda237e38b72e8d8bfc38842fef8753d0b877e305d45.tar "
          },
          "locations": [
            {
              "physicalLocation": {
                "artifactLocation": {
                  "uri": "file://Package-Descriptor"
                }
              }
            }
          ]
        },
        {
          "ruleId": "CVE-2024-4032_python3_0:3.6.15-150300.10.60.1",
          "ruleIndex": 25,
          "level": "note",
          "message": {
            "text": "[CVE-2024-4032] sha256__c202c2f3154b7a3480f4215cb446f9a1ef43affe8ca728bf292317d31cc260ea.tar "
          },
          "locations": [
            {
              "physicalLocation": {
                "artifactLocation": {
                  "uri": "file://Package-Descriptor"
                }
              }
            }
          ]
        }
      ]
    }
  ]
}
